opus/import/obs_cassini_common_pds3.py

```
@@ -0,0 +1,106 @@
+    'F RING': 'SATURN-FRING'
+}
+
+class ObsCassiniCommonPDS3(ObsCommonPDS3, ObsCassiniCommon):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+    ################################################################################
+    # HELPER FUNCTIONS USED BY CASSINI INSTRUMENTS
+    ################################################################################
+    def _cassini_intended_target_name(self):
```

Ruff (0.14.8)

Missing return type annotation for private function `_cassini_intended_target_name`

(ANN202)